.github/workflows/package-bump-pr.yml

```diff
@@ -51,12 +51,25 @@ jobs:
 
       - name: lerna - Bump packages
         run: |
-          npx lerna version --yes --no-push --conventional-commits
+          # bump versions of packages
+          npx lerna version --yes --no-push --conventional-commits -m "chore(publish): publish packages"
           if [[ $(git rev-list --count @{u}..HEAD) -ne 0 ]] ; then
+            # Synchronize @spinnaker/pluginsdk-peerdeps
+            cd packages/pluginsdk-peerdeps
+            yarn sync
+
+            # navigate back to root
+            cd ../..
+
+            # perform any updates to yarn.lock
             yarn
-            git add yarn.lock
-            # Update yarn.lock and fix the lerna commit message (to use conventional commits itself)
-            git commit --amend -m "$(git log -1 --pretty=%B | sed -e 's/^Publish/chore(publish): publish packages/')" yarn.lock
+
+            # commit changes
+            git add yarn.lock packages/pluginsdk-peerdeps
+            git commit -m "feat(peerdep-sync): Synchronize peerdependencies"
+
+            # bump version of @spinnaker/pluginsdk-peerdeps
+            npx lerna version --yes --no-push --conventional-commits -m "chore(publish): publish packages"
           fi
```

Comment:
Is it worth moving this logic into a dedicated script? It seems the other steps have followed that convention, e.g. https://github.com/spinnaker/deck/blob/master/scripts/gha_tag_bumped_packages.sh

Reply:
We could. Is there a standard that is/should be made up to say "If a gha goes beyond x lines change to a script"?

Reply:
Not that I'm aware of, just thinking we may as well be consistent. Given this now has (very simple) bash in it, I think it makes sense to extract it.

Reply:
@mattgogerly Got the script written. Realized that the bumped packages and output changelog needed updating as well. Moved some shared logic to a common script.